Gemfile

```diff
@@ -0,0 +1,13 @@
+source 'https://rubygems.org'
+
+gem 'berkshelf', '~> 4.0'
+gem 'chefspec'
+gem 'chef-sugar'
+gem 'rubocop', '= 0.37.2'
+
+group :integration do
+  gem 'test-kitchen',    '~> 1.7.0'
+  gem 'kitchen-vagrant', '~> 0.19.0'
+  gem 'winrm-transport', '~> 1.0'
+  gem 'winrm-fs'
+end
```

Comment:
TL;DR is we prefer to only use the tools (and exact versions) packaged in ChefDK for development and testing of cookbooks...especially public, community cookbooks @chef-cookbooks/engineering-services maintains.